R/operation.R

```diff
@@ -0,0 +1,30 @@
+
+# These values are used to specify "what will we need for the adjustment?" and
+# "what will we change?". For the outputs, we cannot change the probabilities
+# without changing the classes. This is important because we are going to have
+# to define constrains on the order of adjustments.
+
+input_vals  <- c("numeric", "probability", "class", "everything")
+output_vals <- c("numeric", "probability_class", "class", "everything")
```

Comment:
"everything" was added here for the `mutate()` operation but I don't know if that's the right word.